packages/paste-core/components/radio-button-group/package.json

```diff
@@ -0,0 +1,69 @@
+{
+  "name": "@twilio-paste/radio-button-group",
+  "version": "0.0.0",
+  "category": "user input",
+  "status": "production",
+  "description": "A Radio Button Group is a form element that lets users select a single choice from a list of at least two options.",
```

Comment:
this is the description in the spec: Radio Button Group is a set of related, mutually exclusive options where only one can be selected at a time.

Reply:
updated!